### ISSUE TITLE
FVS: Programmatically add FVS_Cutlist to FVSOUT*.MDB if missing

### ISSUE THREAD

**Issue by lbross** [USFS-PNW/Fia-Biosum-Manager#148]
@sorgtyler had a variant/package in his testing that didn't generate and FVS_Cutlist. This caused exceptions in subsequent processing. He suggests programmatically adding and empty FVS_Cutlist in these cases. @jsfried asked for some additional information about why this happens and why it hasn't been a problem previously.

**Comment by sorgtyler** [comment]
If there are any trees in FVS_TreeInit (even if they’re all dead), an FVS_CutList table is generated by FVS. If they’re all dead, the FVS_CutList table will be empty. If FVS_TreeInit has 0 records, then FVS_CutList is not generated by FVS. 

You can validate issue #148 by deleting all records in FVS_TreeInit while keeping the FVS_StandInit records, then running FVS on the inputs, and lastly running the Append data phase in BioSum’s FVSOUT module. 

More simply, you can delete the cutlist table in an existing FVSOUT database and rerun the FVSOUT Append step.

**Comment by lbross** [comment]
Tested in BlueDemo project where FVS was run prior to implementation of issue #105. Renamed  FVS_CutList for Package 010 to hide it. Ran FVS APPEND process. Received warning message telling me there was no cut list and did I want to continue? Said no the first time and process terminated as expected. Said yes the second time and found an empty FVS_CutList table had been added to replace the one I renamed in the FVSOUT*.MDB. There was also an empty FVS_Tree table in the BM_P010_TREE_CUTLIST.MDB. Closing!
